docs(eslint-plugin): update docs about how to add rules (#4123)
ngrx/platform: projects/ngrx.io/content/guide/eslint-plugin/index.md

Some rules also allow automatic fixes with `ng lint --fix`.

## Adding rules

> Import the plugin via `plugins` or `extends` to setup the rules via the `rules`

Reviewer: ```suggestion
To use the a [rule](#rules), import the NgRx plugin via `plugins` or `extends`.
You can add a rule by adding the rule to the `rules` collection.

```

Author: i agree and added your suggestion in the commit [493e731](https://github.com/ngrx/platform/pull/4123/commits/493e731474ac6431d85a981313cd581dd8ea3c70)

> 

Reviewer: To add better highlighting:

```suggestion
```json
```

```
{
      "plugins": ["@ngrx"],
      "rules": {
        "@ngrx/good-action-hygiene": "error",
      }
}
```
or 

> ```

Reviewer: To add better highlighting:

```suggestion
```json
```

{
      "extends": ["@ngrx/recommended"],
      "rules": {
        "@ngrx/good-action-hygiene": "error",
      }
}
```

> To use a preconfigured configuration, you can add it to `extends`:

Reviewer: I'd like to link to the existing configuration options here:

```suggestion

Instead of manually configuring the rules, there are also [preconfigured configurations](#configurations). To use a configuration, add it to `extends`:

```

> ```

Reviewer: To add better highlighting:

```suggestion
```json
```
